doc/dftb+/manual/dftbp.tex

```diff
@@ -3048,14 +3048,20 @@ \subsubsection{Generalized Born Model}
     \is{FromConstants\cb} requires the dielectric constant as real for \kw{Epsilon},
     the molecular mass in \kw{MolecularMass} \modif{\modtype{mass}} and the
     solvent density in \kw{Density} \modif{\modtype{massdensity}}.
+    \kw{MolecularMass} and \kw{Density} only affect the calculation if
+    ``reference'' is chosen as state of the solvent.
 
-  \item[\is{Shift}] \modif{\modtype{energy}} Shift for free energy calculations.
+  \item[\is{FreeEnergyShift}] \modif{\modtype{energy}} Shift for free energy
+    calculations.
 
-  %\item[\is{Temperature}] Temperature for free energy calculations. Default is
-    %ambient temperature: 298.15 K.
+  \item[\is{Temperature}] Temperature for free energy calculations. Default is
```

Comment:
Need to mention this is always in Kelvin

Reply:
Is there a way to add temperature units to it? I have only found energy units, which might not really be the most suitable choice, unless I can supply the default value + modifier.

Reply:
I don't see a problem with adding a new category for temperature units in prog/dftb+/lib_common/unitconversion.F90

As you convert it by the Boltzmann constant in the parser, isn't it an energy proxy anyway?

Reply:
It would be more convenient to have the temperature in K in the processed input, I find this more intuitive than Hartree. But it is indeed used as energy, I will change it here.